### PR TITLE
<feature> APIGateway - error on missing cert

### PR DESCRIPTION
**Pull request by roleyfoley**
Yell if the certificate can't be found so that cloudformation doesn't have to do it for us

### CODE REVIEW

File: providers/aws/components/apigateway/setup.ftl [hunk 1]
```diff
@@ -458,15 +458,21 @@
                     valueIfTrue(
                         {
                             "RegionalCertificateArn":
-                                getArn(value["domain"].CertificateId, true, regionId),
+                                ( getArn(value["domain"].CertificateId, true, regionId) )?has_content?then(
```
Comment (ml019) [inline providers/aws/components/apigateway/setup.ftl]: contentIfContent(
  getArn(value["domain"].CertificateId, true, regionId) ), 
  "COTFatal: Could not find certificate " + value["domain"].CertificateId
)

means function only called once mostly

File: providers/aws/components/apigateway/setup.ftl [hunk 2]
```diff
@@ -458,15 +458,21 @@
...
+                                ),
                             "EndpointConfiguration" : {
                                 "Types" : [endpointType]
                             }
                         },
                         isRegionalEndpointType,
                         {
                             "CertificateArn":
-                                getArn(value["domain"].CertificateId, true, "us-east-1")
+                                ( getArn(value["domain"].CertificateId, true, "us-east-1") )?has_content?then(
```
Comment (ml019) [inline providers/aws/components/apigateway/setup.ftl]: same as previous content...

File: providers/aws/components/apigateway/setup.ftl (hunk 1 above)
Comment (roleyfoley) [reply providers/aws/components/apigateway/setup.ftl]: Fixed

File: providers/aws/components/apigateway/setup.ftl (hunk 2 above)
Comment (roleyfoley) [reply providers/aws/components/apigateway/setup.ftl]: Fixed

**Pull request merged**